### PR TITLE
run: Move container create to the correct place

### PR DESCRIPTION
**Pull request by qinqon**
The provider container node was created before setting qemu-args correctly

After provisioning the script check-cluster-up.sh is run, this PR
add some logic to check the node and secondary nics existence.

### CODE REVIEW

File: cluster-provision/k8s/check-cluster-up.sh [hunk 1]
```diff
@@ -27,9 +28,18 @@ export KUBEVIRTCI_GOCLI_CONTAINER=kubevirtci/gocli:latest
   bash -x ./cluster-up/up.sh
   ${ksh} wait --for=condition=Ready pod --timeout=200s --all
   ${ksh} wait --for=condition=Ready pod --timeout=200s -n kube-system --all
-  ${ksh} get nodes
+  ${ksh} get nodesç
```
Comment (rmohr) [inline cluster-provision/k8s/check-cluster-up.sh:31]: This line probably causes some issues.

Comment (qinqon) [reply cluster-provision/k8s/check-cluster-up.sh]: Done

**Comment by ormergi** [comment]
Nice!
/lgtm

File: cluster-provision/gocli/cmd/run.go [hunk 2]
```diff
@@ -526,25 +524,25 @@ func run(cmd *cobra.Command, args []string) (retErr error) {
 		}
 
 		if success {
-            // move the VM sound card to a vfio-pci driver to prepare for assignment
-            err = prepareDeviceForAssignment(cli, nodeContainer(prefix, nodeName), soundcardPCIID, "")
-            if err != nil {
-                return err
-            }
+			// move the VM sound card to a vfio-pci driver to prepare for assignment
```
Comment (rmohr) [inline cluster-provision/gocli/cmd/run.go:527]: We should add go-formatting. :/ Can we keep whitespaces?

Comment (rmohr) [reply cluster-provision/gocli/cmd/run.go:527]: Never mind. This was go-fmt changing things :+1:

**Review by rmohr — APPROVED** [review]

File: cluster-provision/gocli/cmd/run.go (hunk 2 above)
Comment (qinqon) [reply cluster-provision/gocli/cmd/run.go:527]: tab is the standard for go, let's try to add those checks at follow up.

Comment (rmohr) [reply cluster-provision/gocli/cmd/run.go:527]: Definitely. :+1:

**Pull request merged**